CruCentralCoast/Community/CommunityGroups/CommunityGroupDetailsVC.swift

```diff
@@ -42,6 +66,19 @@ class CommunityGroupDetailsVC: UIViewController {
             self.movementLabel.text = communityGroup.movement?.name
             self.leaderNamesLabel.text = "Leaders: \(communityGroup.leaderNames ?? "N/A")"
             self.summaryLabel.text = communityGroup.summary
+            //hide the join group button for now
+            self.joinCommunityGroupButton.isHidden = true
```

Comment:
I'd also remove this comment, in general I'd say comments should say why you're doing something, instead of what you're doing because the syntax of swift is already pretty much self documenting regarding the what it's doing